06_deploy_bootstrap_vm.sh

```diff
@@ -42,7 +42,7 @@ $SSH -o ConnectionAttempts=500 core@$IP id
 jq '.nodes[0:3] | {nodes: .}' "${NODES_FILE}" | tee "${MASTER_NODES_FILE}"
 
 # Fix etcd discovery on bootstrap
-add_if_name_to_etcd_discovery "$IP" "eth1"
+# add_if_name_to_etcd_discovery "$IP" "eth1"
```

Comment:
Maybe we should just remove this and the associated function, or if we're leaving it commented add a reference to the mco patch?

Reply:
Lets comment it first, we don't need to wait for MCO to lay files anymore afaik, so it not urgent

Reply:
There's still the patch_node_ignition() stuff

Update the patch to remove add_if_name_to_etcd_discovery() altogether. If we need it back, we can always get it from history